docs/guides/docs/running-a-node/running-a-beta-5-node.mdx

```diff
@@ -0,0 +1,202 @@
+  commentType="{/*"
+/>
+
+<CodeImport
+  file="../installation/index.mdx"
+  comment="install_fuelup_command"
+  commentType="{/*"
+  lang="sh"
+  trim="true"
+/>
+
+<TextImport
+  file="../installation/index.mdx"
+  comment="install_testnet"
+  commentType="{/*"
+/>
+
+<CodeImport
+  file="../installation/index.mdx"
+  comment="install_testnet_command"
+  commentType="{/*"
+  lang="sh"
+  trim="true"
+/>
+
+<TextImport
+  file="../installation/index.mdx"
+  comment="set_default_testnet"
+  commentType="{/*"
+/>
+
+<CodeImport
+  file="../installation/index.mdx"
+  comment="set_default_testnet_command"
+  commentType="{/*"
+  lang="sh"
+  trim="true"
+/>
+
+<TextImport
+  file="../quickstart/building-a-smart-contract.mdx"
+  comment="install_help"
+  commentType="{/*"
+/>
+
+## Getting a Sepolia (Ethereum Testnet) API Key
+
+<TextImport
+  file="./running-a-beta-4-node.mdx"
+  comment="get_ethereum_api_key"
+  commentType="{/*"
+/>
+
+### Infura
+
+<CodeImport
+  file="./running-a-beta-4-node.mdx"
+  comment="infura_key"
+  commentType="//"
+  lang="sh"
+  trim="true"
+/>
+
+### Alchemy
+
+<CodeImport
+  file="./running-a-beta-4-node.mdx"
+  comment="alchemy_key"
+  commentType="//"
+  lang="sh"
+  trim="true"
+/>
+
+<TextImport
+  file="./running-a-beta-4-node.mdx"
+  comment="get_ethereum_api_key_end"
+  commentType="{/*"
+/>
+
+## Generating a P2P Key
+
+<TextImport
+  file="./running-a-beta-4-node.mdx"
+  comment="generate_fuel_p2p_key"
+  commentType="{/*"
+/>
+
+{/*TODO: Double check this is still valid*/}
+
+<CodeImport
+  file="./running-a-beta-4-node.mdx"
+  comment="generate_fuel_p2p_key_command"
+  commentType="//"
+  lang="sh"
+  trim="true"
+/>
+
+<TextImport
+  file="./running-a-beta-4-node.mdx"
+  comment="generate_fuel_p2p_key_end"
+  commentType="{/*"
+/>
+
+## Chain Configuration
+
+{/*TODO: Update link for beta-5*/}
+
+To run a local node with persistence, you must configure a `chainConfig.json` file. Here is the [specific configuration](https://github.com/FuelLabs/fuel-core/blob/v0.20.4/deployment/scripts/chainspec/beta_chainspec.json) for `beta-5`:
+
+<CodeImport
+  file="../../../fuel-core/deployment/scripts/chainspec/beta_chainspec.json"
+  lang="json"
+/>
+
+## Running a Local Node
+
+<TextImport
+  file="./running-a-beta-4-node.mdx"
+  comment="running_a_local_node"
+  commentType="{/*"
+/>
+
+{/*TODO: Update contracts and links for beta-5*/}
+
+```sh
+fuel-core run \
+--service-name {ANY_SERVICE_NAME} \
+--keypair {P2P_SECRET} \
+--relayer {ETH_RPC_ENDPOINT}\
+--ip 0.0.0.0 --port 4000 --peering-port 30333 \
+--db-path  ~/.fuel_beta5 \
+--chain ./chainConfig.json \
+--utxo-validation --poa-instant false --enable-p2p \
+--min-gas-price 1 --max-block-size 18874368  --max-transmit-size 18874368 \
+--bootstrap-nodes /dns4/p2p-beta-4.fuel.network/tcp/30333/p2p/16Uiu2HAm3xjsqASZ68KpaJPkPCMUiMgquhjyDHtxcVxVdFkMgRFf,/dns4/p2p-beta-4.fuel.network/tcp/30334/p2p/16Uiu2HAmJyoJ2HrtPRdBALMT8fs5Q25xVj57gZj5s6G6dzbHypoS \
```

Comment:
```suggestion
--reserved-nodes /dns4/p2p-beta-4.fuel.network/tcp/30333/p2p/16Uiu2HAm3xjsqASZ68KpaJPkPCMUiMgquhjyDHtxcVxVdFkMgRFf,/dns4/p2p-beta-4.fuel.network/tcp/30334/p2p/16Uiu2HAmJyoJ2HrtPRdBALMT8fs5Q25xVj57gZj5s6G6dzbHypoS \
```

Reply:
Also we need to get correct IPs for beta5 network. @tirkesi Could you post here or in the slack what DNS are correct?=)

Reply:
sentry-0 PUBLIC_ADDRESS:  /dns4/p2p-beta-5.fuel.network/tcp/30333/p2p/16Uiu2HAmSMqLSibvGCvg8EFLrpnmrXw1GZ2ADX3U2c9ttQSvFtZX

sentry-1 PUBLIC_ADDRESS: /dns4/p2p-beta-5.fuel.network/tcp/30334/p2p/16Uiu2HAmVUHZ3Yimoh4fBbFqAb3AC4QR1cyo8bUF4qyi8eiUjpVP

---

endpoint that handles the tcp connection: k8s-beta5-beta5tcp-abf39034d2-db60da65b4fcac49.elb.us-east-1.amazonaws.com

```
nc -z -v k8s-beta5-beta5tcp-abf39034d2-db60da65b4fcac49.elb.us-east-1.amazonaws.com 30333                                    
Connection to k8s-beta5-beta5tcp-abf39034d2-db60da65b4fcac49.elb.us-east-1.amazonaws.com port 30333 [tcp/*] succeeded!
```